docs/_rpc/ns-debug.md

```diff
@@ -84,6 +84,25 @@ profile data to disk.
 | RPC     | `{"method": "debug_cpuProfile", "params": [string, number]}` |
 
 
+### debug_dbAncient
+
+Retrieves an ancient binary blob from the freezer. The freezer is a collection of append-only immutable files.
+They include types such as block headers, block bodies, receipts, etc.
+
+| Client  | Method invocation                                           |
+|:--------|-------------------------------------------------------------|
+| Console | `debug.dbAncient(kind string, number uint64)`               |
+| RPC     | `{"method": "debug_dbAncient", "params": [string, number]}` |
+
+### debug_dbAncients
+
+Returns the ancient item numbers in the ancient store.
```

Comment:
"Returns the number of ancient items in the ancient store" reads a bit better :)